### PR TITLE
[Ready for Guardian Review] Add new information for updateMessages API

### PR DESCRIPTION
**Pull request by pjoviak**
Add new information for updateMessages API (edit message)

Issue:
[https://github.ibm.com/toscana/planning/issues/35284](https://github.ibm.com/toscana/planning/issues/35284)  update DX docs for edit message API

@roy713@miguelestrada @jarusso

### CODE REVIEW

File: guides/V1_UpdateMessage.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        id
+        content
+        contentUpdated
+        updated
+        updatedBy {
+          email
+        }
+    }
+  }
+}
```
Comment (roy713) [inline guides/V1_UpdateMessage.md:54]: Two examples here?

Comment (pjoviak) [reply guides/V1_UpdateMessage.md:54]: Yea, can make one.  Thanks

**Review by roy713 — APPROVED** [review]

**Comment by pjoviak** [comment]
@miguelestrada @jarusso  Could you please review?  Thank you

**Comment by pjoviak** [comment]
@miguelestrada @jarusso Could you please review? Thank you

**Comment by pjoviak** [comment]
@miguelestrada @jarusso Could you please review? Thank you

**Comment by miguelestrada** [comment]
@pjoviak Getting this as a first start.  Can you start another PR to update this doc to reflect/explain what happens to the various aspects of the message when it gets updated?  Specifically:
1.  What happens to existing annotations?
2. What happens when an app tries to call this mutation using app as identity vs user?
3. Can an app update a message that it created via generic annotation?

**Pull request merged**